internal/printer/printer_test.go

```diff
@@ -28,6 +28,7 @@ func TestEmit(t *testing.T) {
 		{title: "BooleanLiteral#1", input: `true`, output: `true;`},
 		{title: "BooleanLiteral#2", input: `false`, output: `false;`},
 		{title: "NoSubstitutionTemplateLiteral", input: "``", output: "``;"},
+		{title: "NoSubstitutionTemplateLiteral#2", input: "`\n`", output: "`\n`;"},
```

Comment:
I was mistaken; this test passes even without this PR's change. Add a test which would have failed before this PR's fix.